src/settings.js

```diff
@@ -756,16 +756,6 @@ var EXCEPTION_STACK_TRACES = false;
 // [link]
 var NODEJS_CATCH_EXIT = true;
 
-// Catch unhandled rejections in node. This only effect versions of node older
-// than 15.  Without this, old version node will print a warning, but exit
-// with a zero return code.  With this setting enabled, we handle any unhandled
-// rejection and throw an exception, which will cause  the process exit
-// immediately with a non-0 return code.
-// This not needed in Node 15+ so this setting will default to false if
-// MIN_NODE_VERSION is 150000 or above.
-// [link]
-var NODEJS_CATCH_REJECTION = true;
```

Comment:
Won't removing it like this break existing builds that have `-sNODEJS_CATCH_REJECTION=0`? (I don't remember what happens on unknown settings, but I thought it was an error)

If so, maybe marking it as deprecated would be better.

Reply:
I have marked it as deprecated (see below).